docs/ref/migration-operations.txt

```diff
@@ -269,7 +269,7 @@ be removed (elided) when :ref:`squashing migrations <migration-squashing>`.
 RunPython
 ---------
 
-.. class:: RunPython(code, reverse_code=None, atomic=True, hints=None, elidable=False)
+.. class:: RunPython(code, reverse_code=None, atomic=None, hints=None, elidable=False)
```

Comment:
The existing docs are all still valid for the default atomic migrations.
The semantics only change if one sets atomic to false on the migration -
which is mentioned further down.

Am 03.02.2016 00:32 schrieb "Tim Graham" <notifications@github.com>:
>
> In docs/ref/migration-operations.txt:
>
> > @@ -269,7 +269,7 @@ be removed (elided) when :ref:`squashing migrations
<migration-squashing>`.
> >  RunPython
> >  ---------
> >
> > -.. class:: RunPython(code, reverse_code=None, atomic=True, hints=None,
elidable=False)
> > +.. class:: RunPython(code, reverse_code=None, atomic=None, hints=None,
elidable=False)
>
> I agree the matrix is a bit much. Doesn't the sentence I pointed out need
to be revised though?
>
> —
> Reply to this email directly or view it on GitHub.